app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
Yeah it's interesting to me too. _ tends to just be like used to show something as internal or private, or to avoid usage of this, but the usage is wildly different across languages and such. That it was _role to begin with is more puzzling, since it's definitely not an internal or private variable. Will the switch have any impact on existing data?